### ISSUE TITLE
Locked out of my Android phone

### ISSUE THREAD

**Issue by cafsam** [prey/prey-bash-client#116]
Moto Droid running 2.2.2

Had numeric password protection (PIN) enabled.  As a test, sent the activation SMS and lock.  This worked as my PIN no longer worked.  Then turned off lock and saved changes.  Still wouldn't let me in.  Sent deactivation SMS twice.  Tried to lock with password as my PIN.  Tried lock with different PIN.  Nothing works.  After a few times trying (like 4 or 5) it will give me a message saying that I have incorrectly drawn my unlock pattern 25 or 30 or 35 times, try again in 30 seconds.

Basically I now have a receive-only phone.  What can I do?
